Logger/Employee.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Logger;
+
+public record class Employee(int EID, FullName FName) : Person(FName)
```

Comment:
```suggestion
public record class Employee(int employeeId, FullName fullName) : Person(fullName)
```
Avoid acronyms, unless they are widly used such as `Id`